src/ResourceManager/TrafficManager/Commands.TrafficManager2/help/Add-AzureRmTrafficManagerEndpointConfig.md

```diff
@@ -39,13 +42,28 @@ PS C:\> Set-AzureRmTrafficManagerProfile -TrafficManagerProfile $TrafficManagerP
 
 The first command gets an Azure Traffic Manager profile by using the **Get-AzureRmTrafficManagerProfile** cmdlet.
 The command stores the local profile in the $TrafficManagerProfile variable.
+
 The second command adds an endpoint named contoso to the profile stored in $TrafficManagerProfile.
 The command includes configuration data for the endpoint.
 This command changes only the local object.
+
 The final command updates the Traffic Manager profile in Azure to match the local value in $TrafficManagerProfile.
 
 ## PARAMETERS
 
+### -CustomHeaders
+List of custom header name and value pairs for probe requests.```yaml
```

Comment:
This is one of the trailing yaml string mentioned in the validatehelpissues report.

Reply:
Fixed.